### DISCUSSION TITLE
SmartCookieWeb Preview 16.0

### DISCUSSION THREAD

**Discussion by CookieJarApps** [CookieJarApps/SmartCookieWeb-Preview#97] [General]
Based on Mozac 99.0.7 and GeckoView 99

# SmartCookieWeb Preview 16.0

\+ Fixed purple status bar
\+ Fixed disabling safe browsing not disabling blocking malicious pages

Known bugs:
- Closing a homepage tab whilst another homepage tab is above it doesn't correctly load the homepage - the tab must be clicked on to trigger the homepage

<hr /><em>This discussion was created from the release <a href='https://github.com/CookieJarApps/SmartCookieWeb-Preview/releases/tag/16.0'>SmartCookieWeb Preview 16.0</a>.</em>

**Comment by gsbnlda** [comment]
This file has a virus . https://www.virustotal.com/gui/file/146e5c9f188a41579c012d58e3a5cd05e35576a3213ce514b5108f5f72f7fd44?nocache=1

**Reply by CookieJarApps** [reply]
Hi, unfortunately some of the virus engines (in this case one) on VirusTotal detect false positives on some APKs - a few engines used to flag the WebView-based SmartCookieWeb. The only thing I can do is reach out to the company behind the antivirus flagging it and alert them to the false positive (which I did for SmartCookieWeb, which is now correctly flagged as safe by all engines). I'm not sure how to reach out to the specific company in this case, but I'll look into it. This is definitely a false positive though, which is backed up by how just one engine is flagging the file and the rest mark it as safe.
